snuba/settings/__init__.py

```diff
@@ -35,7 +35,9 @@
             "sessions",
             "transactions",
         },
-        "single_node": True,
+        "single_node": False,
+        "cluster_name": "test_shard_localhost",
+        "distributed_cluster_name": "test_shard_localhost",
```

Comment:
Can we define our own cluster in config rather than rely on this dummy one? This is probably what's causing the tests to fail right now since it's default configuration (localhost:9000) isn't valid in CI.